package.json

```diff
@@ -2,22 +2,25 @@
   "name": "raha-blockchain",
   "version": "0.0.1",
   "description": "Blockchain scripts for Raha.",
-  "main": "built/index.js",
   "author": "Raha Foundation",
   "license": "Apache-2.0",
   "private": false,
   "dependencies": {
     "bs58": "^4.0.1",
     "buffer": "^5.1.0",
+    "firebase": "^4.12.0",
```

Comment:
Thoughts for for future...

Ideally whatever packages `laboratory` depends on should not depend on `firebase` to make it abundantly clear to people that firebase and blockchain are completely separate. Maybe there should be a third `firebase-operations` repo, and both `firebase-operations` and `laboratory` depend on `blockchain`? Also not sure if `blockchain` is the best name since that means so many things to different people, perhaps `immutable-timestamped-data` or something

Reply:
Agreed, I'd like to separate them and their dependencies. Not exactly sure how to do that right now -- like you said, figured we could leave it for a later time :)